src/pages/public/account/LoginPage.vue

```diff
@@ -1,18 +1,23 @@
 <template>
     <div class="container">
         <h1 class="headline">Hi there! <br> <span class="headline--accent">Care to log in?</span></h1>
-        <form-card>
+        <form-card @submit.prevent.native="formSubmit">
             <template #form-controls>
-                <base-input v-model="email" @blur="validateEmail" inputLabel="email" :isValid="isEmailError" placeholder="Enter your email"/>
+                <base-input v-model="email" inputLabel="email" placeholder="Enter your email"/>
                 <password-input v-model="password" />
             </template>
             <template #form-actions>
-                    <base-button block type="button">
-                        <router-link :to="{name: 'register'}">
-                            Register instead
-                        </router-link>
-                    </base-button>
-                    <base-button block type="submit" buttonStyle="primary">Log in</base-button>
+                <base-button block type="button">
+                    <router-link :to="{name: 'register'}">
+                        Register instead
+                    </router-link>
+                </base-button>
+                <base-button block type="submit" buttonStyle="primary">Log in</base-button>
+            </template>
+            <template #form-errors>
+                <base-alert v-if="isLoginError" type="error"  @close="isLoginError = false">
+                    Invalid user credentials. Try again
+                </base-alert>
             </template>
         </form-card>
         <p class="info">Did you forget your password? <router-link to="" class="info--accent">Reset it now</router-link></p>
```

Comment:
It should not but there is no page to reset password. What is better solution? Get rid of it or keep it, maybe something else?

Reply:
If as you say this link isn't working now because there is no page to reset the password then technically in the current state this link is useless for the user until the reset password page will be available, in my opinion, it's better to add it when such page will be available and currently get rid of it or keep it commented with an appropriate comment until you'll add a reset page.